test/credit_card_sanitizer_test.rb

```diff
@@ -102,6 +102,7 @@ class CreditCardSanitizerTest < MiniTest::Test
         assert_nil @sanitizer.sanitize!("blah blah  +4111111111111111.json")
         assert_nil @sanitizer.sanitize!("\"+4111111111111111\"")
         assert_nil @sanitizer.sanitize!("(+4111111111111111)")
+        assert_nil @sanitizer.sanitize!("&#43;1 936 321 1111")
```

Comment:
Hi Cherub,

The number that is being redacted in your example is `431 936 321 1111`, not `1 936 321 1111`. The sanitizer is picking up the HTML entity as part of the credit card number.

```
11007-ggrossman:credit_card_sanitizer ggrossman$ bundle exec irb
irb(main):001:0> require 'credit_card_sanitizer'
=> true
irb(main):002:0> CreditCardSanitizer.new.sanitize!("&#43;1 936 321 1111")
=> "&#43;1 936 ▇▇▇ 1111"
irb(main):003:0> CreditCardSanitizer.new.sanitize!("431 936 321 1111")
=> "431 936 ▇▇▇ 1111"
irb(main):004:0> CreditCardSanitizer.new.sanitize!("1 936 321 1111")
=> nil
irb(main):005:0> 
```